sherpa-onnx/csrc/online-transducer-model.cc

```diff
@@ -115,6 +121,8 @@ std::unique_ptr<OnlineTransducerModel> OnlineTransducerModel::Create(
   switch (model_type) {
     case ModelType::kConformer:
       return std::make_unique<OnlineConformerTransducerModel>(config);
+    case ModelType::kEbranchformer:
```

Comment:
Can you also update
https://github.com/k2-fsa/sherpa-onnx/blob/41d378a128312acc6da09251dc32891a4922f6ee/sherpa-onnx/csrc/online-transducer-model.cc#L199-L200

It is for Android and HarmonyOS.